File: Lecture-6/Code/OpenHashing_Lookup.java
T lookup(int s) {
	int j = 0;
	
	while (@{\color{java_static}t}@[(h(s) - g(s,j)) % @{\color{java_static}m}@] != null) {
		if (@{\color{java_static}t}@[(h(s) - g(s,j)) % @{\color{java_static}m}@].@{\color{java_static}key}@ == s)
			return @{\color{java_static}t}@[(h(s) - g(s,j)) % @{\color{java_static}m}@].@{\color{java_static}value}@
		
		j++;
	}
	
	return null;
}

File: Lecture-6/Code/OpenHashing_Lookup.py
def lookup(s):
	j = 0
	
	while t[(h(s) - g(s, j)) mod m] \
			is not None:
		if t[(h(s) - g(s, j)) mod m][0] != s:
                        j += 1
        if t[(h(s) - g(s, j)) mod m][0] == s:
	        return t[(h(s) - g(s, j)) mod m]
	return None
